src/SeamClient.php

```diff
@@ -31,6 +31,11 @@
 use Seam\Utils\PackageVersion;
 
 use GuzzleHttp\Client as HTTPClient;
+use Seam\SeamHttpApiError;
+use Seam\SeamHttpUnauthorizedError;
+use Seam\SeamHttpInvalidInputError;
+use Seam\SeamActionAttemptFailedError;
+use Seam\SeamActionAttemptTimeoutError;
```

Comment:
For the errors we said we can follow Ruby's example for top level errors because we have the namespace (unless you found this is not appropriate for PHP)


```suggestion
use Seam\HttpApiError;
use Seam\HttpUnauthorizedError;
use Seam\HttpInvalidInputError;
use Seam\ActionAttemptFailedError;
use Seam\ActionAttemptTimeoutError;
```

Reply:
9f4268526d5ac882303a3ef715baba1ba38ac46d